llvm/test/SYCLLowerIR/printf_generic_as_no_opt.ll

```diff
@@ -0,0 +1,580 @@
+  %Id.ascast = addrspacecast %"class.cl::sycl::id"* %Id to %"class.cl::sycl::id" addrspace(4)*
+  %this1 = load %"class.cl::sycl::accessor" addrspace(4)*, %"class.cl::sycl::accessor" addrspace(4)* addrspace(4)* %this.addr.ascast, align 8
+  %0 = bitcast %"class.cl::sycl::id" addrspace(4)* %Id.ascast to %"class.cl::sycl::detail::array" addrspace(4)*
+  %call = call spir_func align 8 dereferenceable(8) i64 addrspace(4)* @_ZN2cl4sycl6detail5arrayILi1EEixEi(%"class.cl::sycl::detail::array" addrspace(4)* align 8 dereferenceable_or_null(8) %0, i32 0) #8
+  %1 = load i64, i64 addrspace(4)* %call, align 8
+  ret i64 %1
+}
+
+; Function Attrs: convergent mustprogress noinline norecurse nounwind optnone
+define linkonce_odr dso_local spir_func %"struct.cl::sycl::detail::AssertHappened" addrspace(1)* @_ZNK2cl4sycl8accessorINS0_6detail14AssertHappenedELi1ELNS0_6access4modeE1025ELNS4_6targetE2014ELNS4_11placeholderE0ENS0_3ext6oneapi22accessor_property_listIJEEEE15getQualifiedPtrEv(%"class.cl::sycl::accessor" addrspace(4)* align 8 dereferenceable_or_null(32) %this) #6 comdat align 2 {
+entry:
+  %retval = alloca %"struct.cl::sycl::detail::AssertHappened" addrspace(1)*, align 8
+  %this.addr = alloca %"class.cl::sycl::accessor" addrspace(4)*, align 8
+  %retval.ascast = addrspacecast %"struct.cl::sycl::detail::AssertHappened" addrspace(1)** %retval to %"struct.cl::sycl::detail::AssertHappened" addrspace(1)* addrspace(4)*
+  %this.addr.ascast = addrspacecast %"class.cl::sycl::accessor" addrspace(4)** %this.addr to %"class.cl::sycl::accessor" addrspace(4)* addrspace(4)*
+  store %"class.cl::sycl::accessor" addrspace(4)* %this, %"class.cl::sycl::accessor" addrspace(4)* addrspace(4)* %this.addr.ascast, align 8
+  %this1 = load %"class.cl::sycl::accessor" addrspace(4)*, %"class.cl::sycl::accessor" addrspace(4)* addrspace(4)* %this.addr.ascast, align 8
+  %0 = getelementptr inbounds %"class.cl::sycl::accessor", %"class.cl::sycl::accessor" addrspace(4)* %this1, i32 0, i32 1
+  %MData = bitcast %union.anon addrspace(4)* %0 to %"struct.cl::sycl::detail::AssertHappened" addrspace(1)* addrspace(4)*
+  %1 = load %"struct.cl::sycl::detail::AssertHappened" addrspace(1)*, %"struct.cl::sycl::detail::AssertHappened" addrspace(1)* addrspace(4)* %MData, align 8
+  ret %"struct.cl::sycl::detail::AssertHappened" addrspace(1)* %1
+}
+
+; Function Attrs: convergent mustprogress noinline norecurse optnone
+define weak_odr dso_local spir_kernel void @_ZTSZZ4mainENKUlRN2cl4sycl7handlerEE_clES2_EUlvE_() #0 comdat !kernel_arg_buffer_location !9 {
+entry:
+  %0 = alloca %class.anon.0, align 1
+  %1 = addrspacecast %class.anon.0* %0 to %class.anon.0 addrspace(4)*
+  call spir_func void @_ZZZ4mainENKUlRN2cl4sycl7handlerEE_clES2_ENKUlvE_clEv(%class.anon.0 addrspace(4)* align 1 dereferenceable_or_null(1) %1) #8
+  ret void
+}
+
+; CHECK: define internal spir_func void @_ZZZ4main{{.*}}
```

Comment:
[`CHECK-LABEL`](https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive)?